### PR TITLE
Implement MDXProvider and add some basic typography theme config

### LINKED ISSUE #98: Style text links
image

They're that gross bright blue underline

### PR DESCRIPTION
**Pull request by robertsoniv**
Also fixed any console errors that were presenting due to bad forward refs or path references (right menu mostly).

#85 - resolved
#98 - resolved

@crhistianramirez @esitarz @djsteinmetz 
This is my first PR to this open source lib so lemme know if I should be doing it differently please!

### CODE REVIEW

**Comment by robertsoniv** [comment]
@crhistianramirez @esitarz I'm going to take a look at some of the design tasks to see if I resolved any of them.

**Comment by robertsoniv** [comment]
@crhistianramirez @esitarz I think I've added the relevant issues to this PR now. Take a gander when you get a moment!

File: content/docs/main-concepts/assignments.mdx [hunk 1]
```diff
@@ -18,6 +18,7 @@ There are basic principles around how assignments work that are critical to unde
 ## Assignments are Inclusive
 
 When a user is created they exist in a vacuum. The user will not have access to any objects until an assignment is made to them directly, or through a higher level party assignment.
+[Test Inner Link]('https://www.google.com')
```
Comment (crhistianramirez) [inline content/docs/main-concepts/assignments.mdx]: Don't think you meant to add this

Comment (robertsoniv) [reply content/docs/main-concepts/assignments.mdx]: Removed :)

**Pull request merged**